packages/main/types/function-calling.ts

```diff
@@ -108,21 +108,21 @@ export enum FunctionDeclarationSchemaType {
 export interface Schema {
   /**
    * Optional. The type of the property. {@link
-   * FunctionDeclarationSchemaType}.
+   * SchemaType}.
    */
-  type?: FunctionDeclarationSchemaType;
+  type?: SchemaType;
   /** Optional. The format of the property. */
   format?: string;
   /** Optional. The description of the property. */
   description?: string;
   /** Optional. Whether the property is nullable. */
   nullable?: boolean;
-  /** Optional. The items of the property. {@link FunctionDeclarationSchema} */
-  items?: FunctionDeclarationSchema;
+  /** Optional. The items of the property. */
+  items?: Schema;
```

Comment:
Yes. It's always been that way in the proto: https://source.corp.google.com/piper///depot/google3/google/ai/generativelanguage/v1main/content.proto;l=489

We moved away from using the generic Schema type in the SDK for function calling because top-level Schemas had different requirements from Schemas nested in `properties` or `items` so we wanted to distinguish and provide more specific guidance for users. It's less clear what the difference is for the top-level responseSchema is so we should leave it as the generic Schema until we have any confirmation that it should be different.

As long as there is a generic Schema, its children should also be generic Schema and not extended versions of itself that other extended versions may not know about.